### DISCUSSION TITLE
Extending library for AWS, Google

### DISCUSSION THREAD

**Discussion by jack0fshad0ws** [RageAgainstThePixel/OpenAI-DotNet#387] [Q&A]
Hi, just a quick question: are there plans to extend the library to also work with providers like AWS or Google Vertex? A unified library to access major providers' APIs could offer a significant advantage over the "official" OpenAI/Azure SDKs etc.
Thanks
Pavel

**Comment by StephenHodgson** [comment]
I do not, and mostly bc I just think that google and these others have their own official sdks.
